meetup/services/generic_api.py

```diff
@@ -0,0 +1,21 @@
+from django.http import HttpResponse
+import requests
+from meetuppizza.settings import base
+
+class GenericApi():
+  BASE_URL = ''
+
+  def __init__(self, link):
+    self.link = link
+
+  def get_response(self):
+    pass
+
+  def build_api_url(self):
+    pass
+
+  def url_exists(self):
```

Comment:
Similarly, this seems to be checking that the entity (a meetup, a yelp business) exists, not that the URL itself exists.